docs/UserGuide.md

```diff
@@ -197,127 +205,139 @@ Deletes a supplier from the list of suppliers in VendorVault.
-Supplier at index 3 is deleted, assuming it existed initially. Otherwise, an error message will be shown.
+- Supplier at index 3 is deleted, assuming there is at least three suppliers displayed. Otherwise, an error message will be shown.
 
 #### Here's how it would look like in the app:
 TO UPDATE IMAGE AFTER FINAL UPDATE TO APPLICATION!!!
 ![delete command](images/deleteSupplierCommand.png)
 
-### Mark a supplier with a status : `mark -s`
+### Marking a supplier : `mark -s`
 
-The `mark` command is used to mark a supplier as either **active** or **inactive**.
-in VendorVault. This helps you keep track of which suppliers are currently active for deliveries and which are not.
+Marks a specified supplier with the specified status in VendorVault.
 
 Format: `mark -s INDEX STATUS`
 
-<box type="details" seamless>
+<box type="info" seamless>
 
 Parameters:
-- `INDEX`: The index of the supplier in the list. Must be a number greater than 0 and must not be blank.
-- `STATUS`: Must be one of the following: `active`, `inactive` and must not be blank.
+- `INDEX`: The index of the supplier to be marked in the displayed list. It must be a number between 1 and the total number of suppliers displayed (inclusive), and cannot be blank.
+- `STATUS`: The suppler status to change to. It must be one of the following values: `active`, `inactive`, and cannot be blank.
 
 </box>
 
-<box type="warning" seamless>
+<box type="caution" seamless>
 
 **Warnings**:
-- At least one space between `mark` and `-s` is needed.
-- At least one space between `-s` and `INDEX` is needed.
-- At least one space between `INDEX` and `STATUS` is needed.
-- Both parameters must be given.
-- `INDEX` and `STATUS` used are case-sensitive.
-- A supplier has a default status of `active`.
+- At least one space is needed between `mark` and `-s`.
+- At least one space is needed between `-s` and `INDEX`.
+- At least one space is needed between `INDEX` and `STATUS`.
+- All parameters must be given.
+- At least one space is needed between parameters.
+- `STATUS` is case-sensitive.
+- An error message will be shown if the user tries to mark a supplier with the same status.
 - A supplier with an `inactive` status signifies that the supplier is not currently active for deliveries.
   However, marking an `active` supplier as `inactive` will not delete the delivery associated with the supplier from the delivery list
   or affect the delivery status in any way.
-</box>
 
-#### Example
-To mark the supplier at index 3 as active:
+</box>
 
-    mark -s 3 active
+Examples:
+- `mark -s 3 inactive`
+- Sorting by name using `sort -s so/a sb/n` followed by `mark -s 1 inactive`
 
-A success message will be displayed if the supplier is successfully marked as active.
+Expected outputs:
+- Supplier at index 3 of the displayed list has status marked as `active`, assuming there is at least three suppliers displayed and the third supplier is not currently `INACTIVE`. Otherwise, an error message will be shown.
```

Comment:
Should be not currently `ACTIVE`, and should the status marked as `active` be in caps as well